packages/components/switch/__tests__/switch.vitest.ts

```diff
@@ -1,10 +1,19 @@
 import { markRaw, nextTick } from 'vue'
 import { mount } from '@vue/test-utils'
-import { describe, expect, test, vi } from 'vitest'
+import { afterEach, describe, expect, test, vi } from 'vitest'
+import { debugWarn } from '@element-plus/utils'
 import { Checked, CircleClose } from '@element-plus/icons-vue'
 import Switch from '../src/switch.vue'
 
+vi.mock('@element-plus/utils/error', () => ({
+  debugWarn: vi.fn(),
+}))
+
 describe('Switch.vue', () => {
+  afterEach(() => {
+    ;(debugWarn as SpyInstanceFn).mockClear()
```

Comment:
We can use `vi.clearAllMocks`

Reply:
Changes are done.

Might be a good idea/practice to set the `clearMocks: true` in the Vitest config 
https://vitest.dev/config/#clearmocks

Reply:
Yeah, good idea. Let's enable it!

Reply:
I just tried `clearMocks: true` and there are 2 failing tests in `form-item.test.ts` which should be fixable but i don't have time at the moment :( Could we merge this PR as is and i'll create another one later?

Reply:
OK！